src/Watchy.cpp

```diff
@@ -9,7 +9,6 @@ RTC_DATA_ATTR BMA423 sensor;
 RTC_DATA_ATTR bool WIFI_CONFIGURED;
 RTC_DATA_ATTR bool BLE_CONFIGURED;
 RTC_DATA_ATTR weatherData currentWeather;
-RTC_DATA_ATTR int weatherIntervalCounter = WEATHER_UPDATE_INTERVAL;
```

Comment:
weatherIntervalCounter needs to be an RTC variable or else it would not be persisted between deepsleep

Reply:
i wondered about that. Thanks for catching it and explaining it.

Reply:
Here's a video I came across that talks about it: https://savjee.be/videos/programming-esp32-with-arduino/using-rtc-memory/
For variables you want to prepend RTC_DATA_ATTR to persist it in RTC memory between deep sleeps, but for constants you won't need to since it's static and stored in flash